### PR TITLE
Rob Wilton #3: clarify "are not ordered" in filter examples

### PR DESCRIPTION
**Pull request by cabo**
→ do not offer `<` comparison

### CODE REVIEW

**Review by gregsdennis — APPROVED** [review]

File: draft-ietf-jsonpath-base.md [hunk 1]
```diff
@@ -1388,16 +1388,16 @@ Comparisons:
 | `$.arr != $.arr` | false | Array comparison |
 | `$.obj == 17` | false | Type mismatch |
 | `$.obj != 17` | true | Type mismatch |
-| `$.obj <= $.arr` | false | Objects and arrays are not ordered |
-| `$.obj < $.arr` | false | Objects and arrays are not ordered |
+| `$.obj <= $.arr` | false | Objects and arrays do not offer `<` comparison |
```
Comment (glyn) [inline draft-ietf-jsonpath-base.md:1391]: Just wondering about the "offering" language. How about the following (and throughout)?
```suggestion
| `$.obj <= $.arr` | false | Objects and arrays cannot be compared using `<` |
```

**Review by glyn — APPROVED** [review]
Suggested wording improvement.

File: draft-ietf-jsonpath-base.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-jsonpath-base.md:1391]: Well, this sounds like a factual statement, but it is not clear if that is a fundamental property (here, it actually pretty much is), or if it is a result of what we decided.

Comment (cabo) [reply draft-ietf-jsonpath-base.md:1391]: We could s/offer/provide for/

Comment (glyn) [reply draft-ietf-jsonpath-base.md:1391]: "provide for" is about the same as "offer" to my taste. How about "do not support"?

Comment (glyn) [reply draft-ietf-jsonpath-base.md:1391]: (On the factual statement point, "not offering" is equally factual - it doesn't say that this spec doesn't offer.)

Comment (cabo) [reply draft-ietf-jsonpath-base.md:1391]: The meaning is the same, the connotations aren't.  I'd like stick with "provide for" if you don't like "offer".

Comment (glyn) [reply draft-ietf-jsonpath-base.md:1391]: It's not a big deal, just a matter of taste. Thanks for considering the alternatives. Merging...

**Pull request merged**